packages/app-store/jitsivideo/lib/VideoApiAdapter.ts

```diff
@@ -11,19 +12,28 @@ const JitsiVideoApiAdapter = (): VideoApiAdapter => {
     getAvailability: () => {
       return Promise.resolve([]);
     },
-    createMeeting: async (): Promise<VideoCallData> => {
+    createMeeting: async (eventData: CalendarEvent): Promise<VideoCallData> => {
       const appKeys = await getAppKeysFromSlug(metadata.slug);
 
-      const meetingID = uuidv4();
+      const meetingPattern = (appKeys.jitsiPathPattern as string) || "{uuid}";
+      const hostUrl = (appKeys.jitsiHost as string) || "https://meet.jit.si/cal";
 
-      // Default Value
-      const hostUrl = appKeys.jitsiHost || "https://meet.jit.si/cal";
+      //Allows "/{Type}-with-{Attendees}" slug
+      const meetingID = meetingPattern
+        .replaceAll("{uuid}", uuidv4())
+        .replaceAll("{Title}", eventData.title)
+        .replaceAll("{Type}", eventData.type)
+        .replaceAll("{Attendees}", eventData.attendees.map((a) => a.name).join("-"))
+        .replaceAll("{Organizer}", eventData.organizer.name)
+        .replaceAll("{Location}", eventData.location || "")
+        .replaceAll("{Team}", eventData.team?.name || "")
+        .replaceAll(" ", "-"); //Last Rule! - Replace all blanks (%20) with dashes;
```

Comment:
<img width="1459" alt="Screenshot 2023-05-19 at 1 23 14 PM" src="https://github.com/calcom/cal.com/assets/1780212/e0dfda66-963b-412c-9d64-caf408115c12">
I would prefer that you reuse these variables and the corresponding logic. That would make it consistent.

Reply:
yea reusing makes sense, tho to be honest jitsi is not a big application / not used much so I don't mind merging whatever works 🙏

Reply:
Alright @hariombalhara ... then let's do it? :)

Reply:
@chief-nerd Let's reuse the existing variable names and if something doesn't exist as a variable then create a new variable name.

I will review it again on Monday